chunkdot/cosine_similarity_top_k.py

```diff
@@ -28,6 +30,8 @@ def cosine_similarity_top_k(
             not marked as available to the OS. In this case is advised to set max_memory
             to chunkdot.utils.get_memory_available at the start of your Python process.
             Default False.
+        show_progress (bool): Whether to show tqdm-like progress bar 
```

Comment:
Use 100 characters in length. Please have a look at the toml tile for the black code style characteristics.